cluster.yml

```diff
@@ -22,7 +22,9 @@ network:
 ssh_key_path: ~/.ssh/test
 ignore_docker_version: false
 
-# Kubernetes authorization mode, currently only `rbac` is supported
+# Kubernetes authorization mode, currently only `rbac` is supported and enabled by default.
+# Use `none` to disable authorization
```

Comment:
none can be treated as absent here. Can we say Use mode: none to disable authorization?